client/modules/wj-util/src/index.ts

```diff
@@ -357,3 +401,15 @@ export function html(strings: TemplateStringsArray, ...subs: (string | string[])
   })
   return toFragment(src)
 }
+
+/**
+ * Performs a modulo operation.
+ * This differs from JavaScript's `%` operator,
+ * which is more of a remainder operator.
```

Comment:
I assume this is with respect to negative values?

Reply:
Correct